util/mutant/__init__.py

```
@@ -0,0 +1,21 @@
+)
+
+__all__ = [
+    "CMutator",
+    "Mutant",
+    "MutationOperator",
+    "ArithmeticOperatorReplacement",
+    "RelationalOperatorReplacement",
+    "LogicalConnectorReplacement",
+    "ConstantReplacement",
+    "ReturnValueReplacement",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)